deps/build.jl

```diff
@@ -70,4 +70,8 @@ provides(BuildProcess, (@build_steps begin
     end
 end), [libfftw, libfftwf])
 
-BinDeps.@install Dict([:libfftw => :libfftw, :libfftwf => :libfftwf])
+if is_windows()
+    BinDeps.@install Dict([:libfftw3 => :libfftw, :libfftw3f => :libfftwf])
```

Comment:
this maps from variable name in this file to variable name in the deps.jl, not library file name at all